### PR TITLE
Use session.count to display number of interventions + load interventions on expand

### PR DESCRIPTION
**Pull request by sbilodeau**

### CODE REVIEW

**Review by randyhoulahan — APPROVED** [review]

**Comment by randyhoulahan** [comment]
it would be nice to load the tab as a route param.
/conferences/sbstta24-sbi3/sbstta-24/documents/:tab
/conferences/sbstta24-sbi3/sbstta-24/documents/statements

File: app/components/meetings/sessions/view.vue [hunk 1]
```diff
@@ -67,14 +69,33 @@ function data(){
...
+      ...session,
+      videos,
+      interventions : null, //Make it reactive
+    }
   });
 
-  this.sessions = sessions.filter(o=>!!o.interventions?.length);
+  if(sessions.length==1) {
+    const [ session ] = sessions; 
+    this.loadInterventions(session._id)
```
Comment (blaisefonseca) [inline app/components/meetings/sessions/view.vue:88]: should this be `const session = sessions[0];`?

**Review by blaisefonseca — COMMENTED** [review]
one comment

File: app/components/meetings/sessions/view.vue (hunk 1 above)
Comment (randyhoulahan) [reply app/components/meetings/sessions/view.vue:88]: potato -  potato.  But if session undefined?

Comment (sbilodeau) [reply app/components/meetings/sessions/view.vue:88]: I used [ ] instead { } so it array deconstruction

**Pull request merged**